docs/src/pages/[platform]/getting-started/migration/migration.vue.mdx

```diff
@@ -1,25 +1,94 @@
 import { Tabs, TabItem } from '@aws-amplify/ui-react';
 import { Alert } from '@aws-amplify/ui-react';
 
+## 2.x to 3.x
+### Installation
+For Vue.js 2 users please continue to use our legacy libary found [here](https://github.com/aws-amplify/amplify-ui/tree/legacy/legacy/amplify-ui-vue).
+
+Install the 3.x version of the `@aws-amplify/ui-vue` library.
+
+<Tabs>
+<TabItem title="npm">
+
+```shell
+npm install aws-amplify @aws-amplify/ui-vue@3.x
+```
+
+</TabItem>
+<TabItem title="yarn">
+```shell
+yarn add aws-amplify @aws-amplify/ui-vue@3.x
+```
+
+</TabItem>
+</Tabs>
+
+### Update and usage
+
+The following breaking changes were introduced in `@aws-amplify/ui-vue` major version `3.0`:
+
+#### 1. Update the handleSignUp function override call
+
+Update the override function call to include the `autSignIn` set to enabled. This must be included if you are overriding `Auth.signUp`.
```

Comment:
Should the conditional of when this patch needs to be applied move to the front?
```suggestion
If you are overriding `Auth.signUp`, update the override function call to include the `autoSignIn` option set to enabled. If this change is not made, customers will not be automatically signed in on signup.
```

Reply:
Updated